src/main/java/kitchenpos/menu/domain/model/Menu.java

```diff
@@ -27,35 +30,30 @@ public class Menu {
     @ManyToOne(fetch = FetchType.LAZY)
     private MenuGroup menuGroup;
 
-    @OneToMany(mappedBy = "menu", cascade = {CascadeType.PERSIST}, orphanRemoval = true)
+    @OneToMany(fetch = FetchType.LAZY, cascade = {CascadeType.ALL}, orphanRemoval = true)
+    @JoinColumn(name = "menu_id", nullable = false)
```

Comment:
```suggestion
    @JoinColumn(name = "menu_id", nullable = false, updatable = false)
```

updatable이 없으니 MenuProduct 수만큼 insert하고 다시 update해주네요...ㅠㅠ
혹시, `updatealbe = false` 안넣으신 이유가 있을까유?!

Reply:
헉 깜빡한 것 같습니다..! 추가하도록 하겠습니다 🙇🏻‍♀️